CSS/style.css

```diff
@@ -413,12 +421,84 @@ img:hover {
   cursor: pointer;
 }
 
-.close {
-  font-size: 45px;
-  font-weight: 600;
+.nav-style > .close {
+  font-size: 60px;
   color: white;
-  margin-top: -300px;
-  float: right;
-  margin-right: 25px;
   cursor: pointer;
+  margin: 20px;
+}
+
+.pop {
+  display: none;
+  position: fixed;
+  margin: 0 auto;
+  top: 0;
+  bottom: 0;
+  max-height: 100vh;
+  width: 100vw;
+  overflow: auto;
+  background: rgb(193 199 208 / 50%);
+  backdrop-filter: blur(3px);
+  padding: 50px 16px;
+}
+
+.pop-content {
+  width: 100%;
+  margin: 0 auto;
+  height: 100vh;
+  background-color: #fff;
+  border: 1px solid #ebecf0;
+  box-sizing: border-box;
+  padding: 20px 16px;
+  border-radius: 15px;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+}
+
+.head-title {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.close-icon {
+  cursor: pointer;
+}
+
+.work-place {
+  margin-top: 22px;
+  margin-bottom: 20px;
+}
+
+.pop-content > ul > li {
+  list-style: none;
+}
+
+.pop-content > ul > li > .year {
+  color: #344563;
+  font-size: 13px;
+  font-weight: 600;
+}
+
+.popimg {
+  position: relative;
+  width: 100%;
+  margin-bottom: 10px;
+  border-radius: 8px;
+}
+
+.popup-buttons {
```

Comment:
- [ ][OPTIONAL] In mobile view, the pop-up seems to be overflowed horizontally. I think this issue is caused by the fixed width assigned to the pop-up buttons. I think using relative measurements  for the width could help solve this issue
![Screenshot 2022-07-27 at 11 53 06](https://user-images.githubusercontent.com/31113416/181205521-b4eafe88-0f53-4c0f-9dbb-2ae2547689c9.png)